brainframe_qt/ui/main_window/activities/stream_configuration/stream_configuration.py

```diff
@@ -699,20 +707,32 @@ def _handle_start_analysis_error(
                 .start()
 
             message_title = self.tr("Active Stream Limit Exceeded")
-            message_desc = self.tr(
+            message = self.tr(
                 "You have exceeded the number of active streams available to "
                 "you under the terms of your license. Consider deleting "
                 "another stream or contacting Aotu to increase your "
                 "active stream limit.")
 
-            BrainFrameMessage.warning(
-                parent=self,
-                title=message_title,
-                warning=message_desc).exec()
+        elif isinstance(exc, bf_errors.StreamNotOpenedError):
+            message = f"<b>{message_desc}</b>" \
+                      f"<br><br>" \
+                      f"{source_hints[stream_conf.connection_type]}<br><br>" \
+                      f"{exc}<br><br>" \
+                      f"{error_text}<b>{exc.kind}</b>"
 
+        elif isinstance(exc, bf_errors.BaseAPIError):
+            message = f"<b>{message_desc}</b>" \
+                      f"<br><br>" \
+                      f"{source_hints[stream_conf.connection_type]}<br><br>" \
+                      f"{error_text}<b>{exc.kind}</b>"
```

Comment:
Why do we add this, esp. without including the `exc` itself in the message. 

We already catch `BaseAPIError` globally: https://github.com/aotuai/brainframe-qt/blob/209cb6fece25f86e6bace152d64b721c294153fa/brainframe_qt/ui/brainframe_app.py#L160-L161

Reply:
I'm confused. This is the error handler for calling api.start_analysis(stream_id). I can offer custom hints there. 

How does your global error handler catch these errors? And why did your previous code catch BaseAPIError on the `_handle_send_stream_conf_error`? I figured it would be important to give clear error messages relating to the PUT /api/streams/{id}/analysis call.